modules/ROOT/pages/comments-callback-mode.adoc

```diff
@@ -1,37 +1,38 @@
-= Configuring the Comments plugin in callback mode
+= Configuring the {pluginname} plugin in callback mode
 :navtitle: Callback mode
 :description: Information on configuring the Comments plugin in callback mode
 :keywords: comments, commenting, tinycomments, callback
 :pluginname: Comments
 :plugincode: comments
 :commentsMode: callback
 
-*Callback mode* is the default mode for the Comments plugin. In the callback mode, callback functions are required to save user comments on a server. The Comments functions (create, reply, edit, delete comment, delete all conversations, resolve, and lookup) are configured differently depending upon the server-side storage configuration.
+*Callback mode* is the default mode for the {pluginname} plugin. In the callback mode, callback functions are required to save user comments on a server. The {pluginname} functions (create, reply, edit, delete comment, delete all conversations, resolve, lookup, and fetch all conversations) are configured differently depending upon the server-side storage configuration.
 
-== Interactive example
-
-The following example uses a simulated server (provided by https://netflix.github.io/pollyjs/[Polly.js]) which has been hidden from the example javascript to keep the example code concise. The interactions between TinyMCE and Polly.js are visible in the browser console.
-
-liveDemo::comments-callback[]
-
-=== How the comments plugin works in callback mode
+== How the {pluginname} plugin works in callback mode
 
 All options accept functions incorporating `+done+` and `+fail+` callbacks as parameters. The function return type is not important, but all functions must call exactly one of these two callbacks: `+fail+` or `+done+`.
 
 * The `+fail+` callback takes either a string or a JavaScript Error type.
 * The `+done+` callback takes an argument specific to each function.
 
-Most (create, reply, and edit) functions require an `+id+` identifying the *current author*.
+Most functions (create, reply, and edit) require an `+id+` identifying the *current author*.
+
+[NOTE]
+**Current author**: The {pluginname} plugin does not know the name of the current user. Determining the current user and storing the comment related to that user is required to be configured by the integrator.
+
+After a user comments (triggering `+tinycomments_create+` for the first comment or `+tinycomments_reply+` for subsequent comments), the {pluginname} plugin requests the updated conversation/s using both `+tinycomments_lookup+` and `+tinycomments_fetch+`, which contains the additional comment with the proper author.
```

Comment:
I don't know exactly how comments use `tinycomments_fetch`. Although I notice that `tinycomments_fetch` is used instead of `tinycomments_lookup` if configured in the initial loading. 

This statement `the {pluginname} plugin requests the updated conversation/s using both `+tinycomments_lookup+` and `+tinycomments_fetch+`` needs to be confirmed.

Reply:
```suggestion
When a user adds a comment (triggering `+tinycomments_create+` for the first comment or `+tinycomments_reply+` for replies), the {pluginname} plugin uses the `+tinycomments_lookup+` callback to retrieve the selected conversation. During the initial load, the `+tinycomments_fetch+` callback retrieves all existing annotated comments, including any additional comments with their respective authors.
```

Reply:
I added a suggestion based on your suggestion @kemister85